WalletWasabi.Fluent/ApplicationStateManager.cs

```diff
@@ -166,10 +168,13 @@ private void CreateAndShowMainWindow()
 		}
 
 		ObserveWindowSize(result, _compositeDisposable);
+		ApplicationViewModel.IsMainWindowShown = false;
 
-		result.Show();
-
-		ApplicationViewModel.IsMainWindowShown = true;
+		if (!_openInBg)
+		{
+			result.Show();
+			ApplicationViewModel.IsMainWindowShown = true;
+		}
```

Comment:
This seems wrong. The method is called `CreateAndShowMainWindow` so it should create and show the MainWindow when it is needed. Otherwise do not have any main window instance. The point of it is to save some memory.

With this change, it will create the main window but won't show it, which is unnecessary.

Reply:
> With this change, it will create the main window but won't show it, which is unnecessary.

My idea was that if we call `result.Show();` here and then outside we fire `Trigger.Hide` there is a brief moment when the wallet pops up, then gets hidden.

https://user-images.githubusercontent.com/54551879/219621842-72c6ec26-f167-444f-9abf-c558406d1e36.mp4

With this approach I wanted skip show the very first time.

Reply:
Simplifyed it